src/core/fonts.js

```diff
@@ -212,6 +215,8 @@ class Glyph {
     this.operatorListId = operatorListId;
     this.isSpace = isSpace;
     this.isInFont = isInFont;
+    this.isWhitespace = isWhitespace(unicode);
```

Comment:
Given the existing `isSpace` property, could we perhaps call this new one e.g. `isUnicodeSpace` (or something) instead to more clearly distinguish them from each other?

Also, please update the name of the new helper function accordingly to avoid confusion with this already existing utility function:
https://github.com/mozilla/pdf.js/blob/8ac0ccc2277a7c0c85d6fec41c0f3fc3d1a2d232/src/core/core_utils.js#L247-L250